files/build_templates/docker_image_ctl.j2

```diff
@@ -259,6 +259,16 @@ function postStartAction()
             # This flag will be set to "1" after DB migration/initialization is completed as part of config-setup
             $SONIC_DB_CLI CONFIG_DB SET "CONFIG_DB_INITIALIZED" "0"
         else
+            if [[ "$BOOT_TYPE" != "warm" ]]; then
```

Comment:
Why is this setting needed now, when it was not needed earlier? And, why is warm-reboot an exception?

Reply:
Because db_migrator busy waits when the CONFIG_DB_INITIALIZED flag is not set. Probably because of ConfigDBConnector https://github.com/sonic-net/sonic-swss-common/blob/master/common/configdb.cpp#L29.

I've only checked for warm-reboot since warm-reboot might already contain the  CONFIG_DB_INITIALIZED from the previous db.

I feel its better to update db_migrator to not depend on CONFIG_DB_INIT, what do you think?